server/chat-plugins/randombattles/index.ts

```diff
@@ -846,6 +855,9 @@ export const commands: Chat.ChatCommands = {
 		}
 
 		if (!movesets.length) {
+			if (isInexact) {
+				return this.errorReply(`Error: No Pok\u00e9mon named '${args[0].trim()}' was found. (Check your spelling?)`);
+			}
 			return this.errorReply(`Error: ${species.name} has no Random Battle data in ${format.name}`);
 		}
 		let buf = movesets.join('<hr/>');
```

Comment:
Unlike the fuzzy searching from the /dt command, I opted not to notify the user that nothing was found under the exact spelling of the argument.  If this is desired, it should likely be done here.